qiskit_rng/generator.py

```diff
@@ -181,8 +181,8 @@ def _run_circuits(
                                        job_tags=[self._job_tag], memory=True)
             logger.info("Jobs submitted to %s. Job set ID is %s.", self.backend, job.job_set_id())
         else:
-            job = self.backend.run(assemble(transpiled, backend=self.backend, shots=shots,
-                                            memory=True, noise_model=self.noise_model))
+            job = self.backend.run(transpiled, shots=shots, memory=True,
+                                   noise_model=self.noise_model)
```

Comment:
This is actually slightly more complicated than just passing in circuits. Since V1 backend has not been removed (only deprecated) in `qiskit-terra`, we'll need to continue to support that. V1 backend expects Qobj while V2 backend expects circuits. So this should be some like

```
if isinstance(self.backend, IBMQBackend):
    use job manager
elif isinstance(self.backend, BaseBackend):
    pass in qobj
else:
    pass in circuits
```